index.html

```diff
@@ -55,6 +55,189 @@ <h2 class="headline-text">
         </li>
     </ul>
     <div class="background-img"></div>
-  </section>  
+  </section>
+  <!--middle section-->
+  <section>
+    <div class="middle-page">
+        <div>
+            <h2 class="recent-work">
+                My Recent Works
+            </h2>
+            <hr class="line">  
+        </div>
+        <div>
+            <img src="top-image.png" alt="middle-page-imag" width="327px" height="237px">
+            <h3 class="multi-post">Multi-Post Stories</h3>
+            <p class="middle-page-discription">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="language1">
+                <li>
+                    <button class="lang-button">CSS</button>
+                </li>
+                <li>
+                    <button class="lang-button">HTML</button>
+                </li>
+                <li>
+                    <button class="lang-button">Bootstrap</button>
+                </li>
+                <li>
+                    <button class="lang-button">Ruby</button>
+                </li>
+            </ul>
+            <button class="see-project1">See Project</button>
+        </div>
+        <div class="pages-container">
+            <div class="page">
+                <h4 class="page-header">Profesional Art Printing Data</h4>
+                <p class="page-paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="page-language">
+                    <li>
+                        <button class="page-lang-button">HTML</button>
+                    </li>
+                    <li>
+                        <button class="page-lang-button">Bootstrap</button>
+                    </li>
+                    <li>
+                        <button class="page-lang-button">Ruby</button>
+                    </li>
+                </ul>
+                <button class="page-see-project">See Project</button>
+            </div>
+            <div class="page">
+                <h4 class="page-header">Profesional Art Printing Data</h4>
+                <p class="page-paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="page-language">
+                    <li>
+                        <button class="page-lang-button">HTML</button>
+                    </li>
+                    <li>
+                        <button class="page-lang-button">Bootstrap</button>
+                    </li>
+                    <li>
+                        <button class="page-lang-button">Ruby</button>
+                    </li>
+                </ul>
```

Comment:
[OPTIONAL]
- [ ] Kindly you may consider removing your tech stacks in the `button` and using only the generic `li` since the `button` don't have any semantic meaning here.